sdk/include/reactos/shlguid_undoc.h

```diff
@@ -127,7 +127,6 @@ DEFINE_GUID(CLSID_InboundConnectionUi,     0x7007ACC3, 0x3202, 0x11D1, 0xAA, 0xD
 DEFINE_GUID(CLSID_InternetConnectionUi,    0x7007ACC4, 0x3202, 0x11D1, 0xAA, 0xD2, 0x00, 0x80, 0x5F, 0xC1, 0x27, 0x0E);
 DEFINE_GUID(CLSID_LanConnectionUi,         0x7007ACC5, 0x3202, 0x11D1, 0xAA, 0xD2, 0x00, 0x80, 0x5F, 0xC1, 0x27, 0x0E);
 DEFINE_GUID(CLSID_VpnConnectionUi,         0x7007ACC6, 0x3202, 0x11D1, 0xAA, 0xD2, 0x00, 0x80, 0x5F, 0xC1, 0x27, 0x0E);
-#define CLSID_ConnectionFolder CLSID_NetworkConnections
```

Comment:
In view of our previous discussion:
https://github.com/reactos/reactos/pull/7503#issuecomment-2481319399
https://github.com/reactos/reactos/pull/7503#issuecomment-2481362663
https://github.com/reactos/reactos/pull/7503#issuecomment-2481382654
I wonder whether it would be fine to keep this define (in the undoc header) to serve as documentation purposes.

Reply:
The Win98 one is probably for some kind of compatibility shim, until we find out how it's used we don't really have a need for it.

Reply:
I reverted the aliased CLSID removal.